src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 20px;
+  background-color: black;
+}
+
+.inner,
+.wrapper,
+p {
+  border: 1px solid #e75551;
+  padding: 10px;
+}
+
+.wrapper {
+  width: 328px;
+  background-color: #308dc0;
+}
+
+.wrapper:nth-child(2) {
+  margin: 40px 0;
```

Comment:
That's not what I expected from you.

Remove `.wrapper:nth-child(2)` and add `margin` to `.wrapper`. `body { margin: 0;}`
If you don't understand what I want, write to me in Slack.